Sources/Logging/TestingLogHandler.swift

```diff
@@ -0,0 +1,298 @@
+import Foundation
```

Comment:
(nitpick before we merge this eventually someday: needs license header)

Reply:
Done.